### PR TITLE
SAAS-781 - password reset

### PR DESCRIPTION
**Pull request by dpsommer**

### CODE REVIEW

File: airtime_mvc/application/views/scripts/login/password-restore.phtml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <div class="login_box">
   <div class="logobox">&nbsp;</div>
   <h2><?php echo _("Reset password") ?></h2>
-  
+
   <div id="login" class="login-content clearfix">
     <p class="light">
-        <?php echo _("Please enter your account e-mail address. You will receive a link to create a new password via e-mail.")?>
+        <?php echo _("Enter your account e-mail address or your username (or both, if you have multiple accounts
+                      using the same email address), and we'll send you a link to reset your password.")?>
```
Comment (asantoni) [inline airtime_mvc/application/views/scripts/login/password-restore.phtml:8]: If the username is optional, then the e-mail field should be first in the form, not the username field. (Right??)

Comment (asantoni) [inline airtime_mvc/application/views/scripts/login/password-restore.phtml:8]: Just as an aside, we really need to force each user to have a unique email address. The way this works works is super confusing. Maybe we can tackle this when we do invitation emails.

File: airtime_mvc/application/controllers/LoginController.php [hunk 2]
```diff
@@ -168,15 +165,14 @@ public function passwordRestoreAction()
                 if ($success) {
                     $this->_helper->redirector('password-restore-after', 'login');
                 } else {
-                    $form->email->addError($this->view->translate(_("Email could not be sent. Check your mail server settings and ensure it has been configured properly.")));
+                    $form->email->addError($this->view->translate(_("There was a problem sending the recovery email.")));
                 }
             } else {
-                $form->email->addError($this->view->translate(_("Given email not found.")));
+                $form->email->addError($this->view->translate(_("We couldn't find the email you entered - you can also try <a href='https://account.sourcefabric.com/pwreset.php'>here</a>.")));
```
Comment (asantoni) [inline airtime_mvc/application/controllers/LoginController.php]: Please make this URL a constant in constants.php or where ever our branding constants are.

**Comment by asantoni** [comment]
Please let me know after you fix up those couple of things and the merge conflicts.

**Comment by dpsommer** [comment]
Done - agreed that we should force unique emails at some point, though I think we still have to handle this case for backwards compatibility either way.

**Pull request merged**